bibliopixel/colors/wheel.py

```diff
@@ -15,8 +16,9 @@ def _gen_wheel():
             b = 127 - p % 128
             r = p % 128
             g = 0
+        result.append((r, g, b))
 
-    return r, g, b
+    return result
```

Comment:
Hahaha... yeah, at least. I really think there was just the one animation that used it.